packages/cli/src/InternalHooks.ts

```diff
@@ -547,75 +547,38 @@ export class InternalHooks {
 		});
 	}
 
-	/**
-	 * Credentials
-	 */
-
-	async onUserCreatedCredentials(userCreatedCredentialsData: {
-		user: User;
-		credential_name: string;
-		credential_type: string;
-		credential_id: string;
-		public_api: boolean;
+	async onLdapSyncFinished(data: {
```

Comment:
I think you're accidentally re-adding the LDAP stuff removed in this PR: https://github.com/n8n-io/n8n/pull/10157/files